### PR TITLE
Add new extension: Flaticon — Search Icons

### PR DESCRIPTION
**Pull request by kudrykv**
## Description

Add a new extension that allows searching for icons on the [flaticon.com](flaticon.com)

## Screencast

https://user-images.githubusercontent.com/12697803/202748313-8c5b15c7-3c9e-4192-b526-f67ad4268bc4.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @kudrykv 👋 

Seems like you placed your extension in the wrong folder (/extensions/flaticon/flaticon), could you move it out to /extensions/flaticon? 😊

**Comment by kudrykv** [comment]
@pernielsentikaer whoops. Fixed!

**Comment by kudrykv** [comment]
@pernielsentikaer could you please look into this PR?

File: extensions/flaticon/src/commandSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export default () => {
+  const [search, setSearch] = useState('');
+  const auth = onAuth(apiKey);
+  const results = onSearchIcons(auth.token, search);
+
+  const isLoading = auth.isLoading || results.isLoading;
+  const list = results.data?.list || [];
+
+  return (
+    <Grid isLoading={isLoading} onSearchTextChange={setSearch} throttle>
```
Comment (pernielsentikaer) [inline extensions/flaticon/src/commandSearch.tsx:23]:
```suggestion
    <Grid isLoading={isLoading} onSearchTextChange={setSearch} inset={Grid.Inset.Medium} columns={8} throttle>
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Sorry that it slipped, I tested the extension, and it works as expected - I think I would add some inset and columns like suggested 😊

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**